### ISSUE TITLE
Prettier's Function Parameters Syntax Highlighting

### ISSUE THREAD

**Issue by RyanLarge13** [Acode-Foundation/Acode#678]
I would love to see Syntax highlighting for function parameters when the variables are being used within the functions code block!!

**Comment by deadlyjack** [comment]
I don't understand your suggestion can you give examples or something?

**Comment by RyanLarge13** [comment]
Yes!! Forgive me for the late reply. I will give you a great example on VS code and compare it with Acode for better understanding.

In this photo below I have a basic run function that takes to parameters highlighted in red given my theme, but when the parameter variables are called within the return they are not highlighted in red, it shows as plane text as when calling any variable. This is the Acode example
![Acode](https://github.com/deadlyjack/Acode/assets/101679941/3f6d806f-c67b-4c7a-b5bc-370ead1602df)

In this photo below I have the same exact function being called and executed with the same parameters, this is in VS code. As you can see, the parameters are highlighted in orange and when they are called in the return statement, these parameter variables are also highlighted in orange indicating that you are using the variables you defined in the functions parameters.
![VScode](https://github.com/deadlyjack/Acode/assets/101679941/bd4941f4-6908-4336-8af1-3bdfbb11f7e1)

I think this is very handy and would be a great addition to the text editor!
